packages/ses/README.md

```diff
@@ -27,34 +27,79 @@ npm install ses
 
 ## Usage
 
-### Module
+### Lockdown
 
-This example locks down the current realm, turning it into a starting
-compartment.
-Within a compartment, there is a `Compartment` constructor that conveys
-"endownments" into the new compartment's global scope, and a `harden` method
-that that object and any object reachable from its surface.
-The compartment can import modules and evaluate programs.
+SES introduces the `lockdown` function.
+Alters the surrounding execution environment such that no object intrinsically
+accessible, like the array or function prototypes (`[].prototype` or `(() =>
+0).prototype`, can be replaced or subverted to attack other code in the same
+execution environment; and tames other intrinsically accessible utilities like
+regular expressions, random numbers, and clocks, to minimize opportunities for
+malicious programs to covertly infer the behavior of other programs running in
+the same execution environment.
 
 ```js
-import {lockdown} from "ses";
+import 'ses';
+
+import 'my-vetted-shim';
+
+lockdown();
+
+console.log(Object.isFrozen([].prototype));
+// true
+```
+
+### Harden
+
+SES introduces the `harden` function.
+*After* calling `lockdown`, the `harden` function ensures that whatever object
+you give it is also transitively frozen out to the execution environments
+intrinsically available objects.
+This allows mutually suspicious programs in the same execution environment to
+share these hardened objects knowing that the other party is limited to
+interacting with the functions expressly given and cannot mutate their surfaces
+
+```js
+import 'ses';
 
 lockdown();
 
+const capability = harden({
+  use() {}
+});
+
+console.log(Object.isFrozen(capability));
+// true
+console.log(Object.isFrozen(capability.use));
+// true
+```
+
+### Compartment
+
+SES introduces the `Compartment` constructor.
+A compartment is an evaluation and execution environment with its own
+`globalThis` and wholly independent system of modules, but otherwise shares
+the same batch of intrinsics like `Array` with the surrounding compartment.
+The concept of a compartment implies the existence of a "start compartment",
+the initial execution argument.
+
+```js
+import 'ses';
+
 const c = new Compartment({
     print: harden(console.log),
 });
 
 c.evaluate(`
-    print("Hello! Hello?");
+    print('Hello! Hello?');
 `);
 ```
 
 The new compartment has a different global object than the start compartment.
 The global object is initially mutable.
 Locking down the start compartment hardened many of the intrinsics in global
```

Comment:
Avoid the phrase "locking down the start compartment". This invites the confusion the rest of the paragraph corrects. `lockdown` locks down the realm, and therefore the shared intrinsics, that the start compartment shares with these other compartments.